mlrun/api/utils/projects/consumers/nuclio.py

```diff
@@ -45,11 +64,10 @@ def delete_project(self, session: sqlalchemy.orm.Session, name: str):
 
     def get_project(
         self, session: sqlalchemy.orm.Session, name: str
-    ) -> mlrun.api.schemas.ProjectOutput:
-        response = self._send_request_to_api("GET", f"projects/{name}")
+    ) -> mlrun.api.schemas.Project:
+        response = self._get_project_from_nuclio(name)
```

Comment:
What happens if project does not exist in Nuclio? From browsing the code it seems that it will raise an exception - is that what we expect in this flow?

Reply:
Yes, it will raise.
Generally GET detail (one resource not list) should return error on not found.
So same here - we do want it in this flow.
Note that GET operations on the manager are not proxied to the consumers, but the manager simply read from master, so practically today no one calls `get_project` on nuclio